### PR TITLE
fix: lower protobuf hash order of magnitude for PHP

### PR DESCRIPTION
**Pull request by bshaffer**
Temporary fix for https://github.com/protocolbuffers/protobuf/issues/8114

### CODE REVIEW

File: src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java [hunk 1]
```diff
@@ -352,11 +352,13 @@ private void readResources(Document document) {
 
         if (method.request() != null) {
           Message request = allMessages.get(method.request().reference());
-          String requestFieldName =
-              Name.anyCamel(request.getName(), "resource").toLowerUnderscore();
-          input.getFields().add(new Field(requestFieldName, request, false, null, null));
-          methodHttpOption.getProperties().put("body", requestFieldName);
-          methodSignatureParamNames.put("", requestFieldName);
+          if (request != null) {
```
Comment (vam-google) [inline src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java:355]: car request be null? if yes, under which circumstances? (it was not required for the whole `compute.proto` before so I'm wondering why it is needed now

**Review by vam-google — APPROVED** [review]
LGTM with a question

**Comment by vam-google** [comment]
@bshaffer Please add a TODO item to revert it to 2^29 with a link to the protobuf bug.

File: src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java (hunk 1 above)
Comment (bshaffer) [reply src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java:355]: I received a null pointer exception for the first time today when using the ignorelist, and I thought this fixed it but I'm still having an issue, so I'll revert this change and submit in a different PR

Comment (bshaffer) [reply src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java]: (reverted)

**Pull request merged**